sig-cli/CONTRIBUTING.md

```diff
@@ -206,6 +183,8 @@ A feature request is an [issue] mentioning `@kubernetes/sig-cli-feature-requests
 To encourage readership, the issue description should _concisely_ (2-4 sentence) describe
 the problem that the feature addresses.
 
+
+//Absolutelly not sure about these 2 next sections. Feel super heavyweight though. Is this the way the SIG is working actually? 
```

Comment:
Not really.

I think this can probably be simplified, and some of the links are no longer relevant.

What would the *actual* process be for a new feature?  Here is my attempt:

1. Open an issue on the k/kubectl repo using "Enhancement Request" (kind/feature label)

2. The monthly bug scrub meeting will triage the issue, which can result in one of the following outcomes:
    a. Issue gets triage/accepted, meaning it is something that the SIG would consider a PR for
    b. Issue is closed, and an explanation is given why the feature is not something that a PR would be accepted for at this time
    c. More information is requested
    d. You are asked to attend a bi-weekly SIG meeting to discuss the feature before moving ahead.

     (2a. Alternatively, you can expedite this process by attending a bi-weekly SIG meeting and put an item on the agenda to discuss the Enhancement Request issue you created)

I don't know if it makes sense to talk about the general PR workflow here or not, because this is not CLI specific, but:

3. You work on the feature and submit a pull request to the k/k repo when you are done.

4. Reviewers will review the PR, either requesting changes or approving and merging it.  This process can take some time, so be patient.